### DISCUSSION TITLE
How to display footer on the home page?

### DISCUSSION THREAD

**Discussion by heihahehe** [thuliteio/doks#1157] [Q&A]
How to display footer on the home page?
![20240101233423](https://github.com/gethyas/doks/assets/152707762/61ff55d6-2d07-4c6a-8ee5-a06c5f2ba5f7)

**Comment by h-enk** [comment]
The footer is displayed on the homepage per default — see the [default Doks](https://doks.netlify.app/)

Your screenshot shows my customization for [getdoks.org](https://getdoks.org/)

**Reply by heihahehe** [reply]
But installing a new doks also does not display the footer, may I ask where the setting is incorrect. Same thing as getdoks.org.

**Reply by h-enk** [reply]
Can't replicate? I ran:

```bash
npm create hyas@latest doks-1157 -- --template doks && cd doks-1157 && npm install && npm run dev
```

And the footer shows:

![Snag_685dc999](https://github.com/gethyas/doks/assets/3902872/322cdd22-0d69-40c8-9bd3-9ca9f34d9cac)

**Reply by heihahehe** [reply]
I found the cause of the problem in "_custom.scss".

.home .footer {
@include visually-hidden;
}

After deletion, the footer can be displayed.
![105](https://github.com/gethyas/doks/assets/152707762/6f53520e-5caa-4fc3-a9e9-9c5741222014)
